X16 Reference - 06 - Math Library.md

```diff
@@ -25,7 +25,7 @@ that is inside BASIC, and so it requires much of the BASIC zeropage to be intact
 The reason is that that routine ultimately relies on some internal BASIC routines that use a lot of BASIC zero page space.
 Ideally in the future, the `VAL_1` routine gets a new implementation that doesn't rely on the code in BASIC, thereby removing this restriction.
 
-#### X16 Additions
+**X16 Additions**
```

Comment:
I'd be wary of removing heading semantics (two other instances on this page). But I recognise h3 might look a bit big here.

Reply:
The problem with the headers is that Markdown creates anchor elements, and when you have two identical headers, you have two identical anchors. 

 I made the choice to go with bold there so as not to have two instances of `<a href=#x16-additions">...</a>`, I chose to demote that to a bolded topic.

Reply:
I see. The second one becomes `#x16-additions-1` on GitHub, but that might not be true everywhere and is prone to breaking if the document is so much as reordered. They can be customised like `<h2 name="alice">Bob</h2>` but it's certainly not as nice once you start mashing HTML into it.